config/redisConfig.js

```diff
@@ -0,0 +1,25 @@
+const redis = require("redis");
+
+
+var Connect = function () {
+  var client = redis.createClient({
+    url: process.env.REDIS_URL,
+    socket: {
+      connectTimeout: 30000
```

Comment:
@jayesh9747  thi s time out will be configurable in env. Add this in envVariables